src/VisualBoard.ts

```diff
@@ -1,18 +1,36 @@
 
-export function render(board: Array<Array<number>>) {
+export function buildGraphicalBoard(board: Array<Array<number>>): String {
```

Comment:
I'd name it renderBoard, or renderToString, but 'build' doesn't carry much of what the function is doing.